app/Filament/Resources/NodeResource/RelationManagers/AllocationsRelationManager.php

```diff
@@ -122,8 +132,19 @@ public function table(Table $table): Table
 
                                     $start = max((int) $start, 0);
                                     $end = min((int) $end, 2 ** 16 - 1);
-                                    foreach (range($start, $end) as $i) {
-                                        $ports->push($i);
+                                    $range = $start <= $end ? range($start, $end) : range($end, $start);
```

Comment:
If the first port number is greater than the second port number, then no numbers should be added.